internal/notif/ntfy/client.go

```diff
@@ -64,12 +64,14 @@ func (c *Client) Send(entry model.NotifEntry) error {
 		Title    string   `json:"title"`
 		Priority int      `json:"priority"`
 		Tags     []string `json:"tags"`
+		Markdown bool     `json:"markdown"`
 	}{
 		Topic:    c.cfg.Topic,
 		Message:  string(body),
 		Title:    string(title),
 		Priority: c.cfg.Priority,
 		Tags:     c.cfg.Tags,
+		Mardown:  true,
```

Comment:
This is a typo. When you make changes, make sure tests pass.
```suggestion
		Markdown:  true,
```

Reply:
Fixed